src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,44 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public class DeleteCommand extends Command {
+
+    private final int index;
+
+    private static final String NO_INDEX_ERROR = "Please specify a duke.task to delete!";
+    private static final String INDEX_NOT_INT_ERROR = "Please specify a number instead!";
+    private static final String OUT_OF_BOUND_ERROR = "Not in the list!";
+
+    public DeleteCommand(String index) throws DukeException {
+        if (index.isBlank()) {
```

Comment:
Would it be better to indicate that the two `index`'s here are of different types? I was a little confused as the index which is the class attribute was of `int`.

Maybe it would be better to specify something like `indexString` in order to distinguish the two.

> Note: Similar for the other classes using `index`